src/main/java/com/iexec/core/replicate/ReplicateModel.java

```diff
@@ -38,7 +38,7 @@ public class ReplicateModel {
     private String resultLink;
     private String chainCallbackData;
     private String contributionHash;
-    private String appStdout;
+    private String appLogs;
```

Comment:
```suggestion
    private ComputeLogs appLogs;
```

Reply:
FYI: `appLogs` is the link to the logs, not the actual logs. So it should stay as a String :)

Reply:
Totally right! Thanks!